src/executor/Executor.cxx

```diff
@@ -286,14 +286,13 @@ void *ExecutorBase::entry()
     {
         Executable *msg = nullptr;
         unsigned priority = UINT_MAX;
-        long long wait_length = activeTimers_.get_next_timeout();
-        if (!selectPrescaler_ || empty()) {
+        if (!selectPrescaler_ || ((msg = next(&priority)) == nullptr)) {
```

Comment:
Should we cleanup the style with hanging braces while we are making changes here?

Reply:
done